src/tests/nativeaot/SmokeTests/HelloWasm/HelloWasm.cs

```diff
@@ -1705,20 +1700,6 @@ private static void TestStaticPInvokeOverloadedInDifferentModules()
         StartTest("Static PInvoke of overloaded function in different modules test");
         EndTest(CommonFunctionNameInModule1(12) == 12 && CommonFunctionNameInModule2(34) == 34);
     }
-
-    [DllImport("StaticModule1", EntryPoint = "StaticIncompatFunctionName")]
-    private static extern int CallAbiIncompatFunctionWithInt(int arg);
-
-    [DllImport("StaticModule2", EntryPoint = "StaticIncompatFunctionName")]
-    private static extern void CallAbiIncompatFunctionWithFloat(float arg);
-
-    // Compilation should produce a warning for CallAbiIncompatFunctionWithInt and CallAbiIncompatFunctionWithFloat.
-    [System.Runtime.InteropServices.UnmanagedCallersOnly(EntryPoint = "JustForRooting2")]
-    private static void RootFuncDup()
-    {
-        CallAbiIncompatFunctionWithInt(0);
-        CallAbiIncompatFunctionWithFloat(1);
-    }
```

Comment:
It turns this part duplicates what's tested below with `TestNativeCallsWithMismatchedSignatures`, so I took it out.